docs/guide/content_rendering/embed_and_list_content/list_products_in_catalog.md

```diff
@@ -0,0 +1,59 @@
+---
+description: List all products contained in a predefined catalog.
+---
+
+# List products in catalog
+
+To list products from a specific [catalog](../../catalog/catalog.md#catalogs), you can either use the Content query Field,
+or a custom controller.
+
+## List products with Content query Field
+
+To list products from a catalog in a [Content query Field](../../../api/field_types_reference/contentqueryfield.md),
+add this Field to a Content Type definition.
+
+Select the ["Catalog" Query type](../queries_and_controllers/built-in_query_types.md#catalog) and "Product" as returned type.
+In parameters, provide the identifier of the catalog.
```

Comment:
```suggestion
Select the ["Catalog" Query type](../queries_and_controllers/built-in_query_types.md#catalog).
In parameters, provide the identifier of the catalog.
```

Currently there is not valid option here for 4.x as new PC works on multiple content types. Mentioned "Product" refer to deprecated 3.3 product catalog.